schema-util/bigquery/src/main/java/io/apicurio/registry/rules/validity/BigqueryContentValidator.java

```diff
@@ -0,0 +1,23 @@
+package io.apicurio.registry.rules.validity;
+
+import io.apicurio.registry.content.ContentHandle;
+import io.apicurio.registry.rules.BigqueryGsonBuilder;
+import io.apicurio.registry.rules.RuleViolationException;
+import io.apicurio.registry.types.RuleType;
+
+import java.util.Map;
+
+public class BigqueryContentValidator extends BigqueryGsonBuilder implements ContentValidator {
+
+    @Override
+    public void validate(ValidityLevel level, ContentHandle artifactContent, Map<String, ContentHandle> resolvedReferences)
+            throws RuleViolationException {
+        try {
+            parseFields(artifactContent.content());
```

Comment:
Validity implementation does not take `ValidityLevel` into account. Most importantly, this means that validity rule is always applied.